cumulusci/robotframework/CumulusCI.py

```diff
@@ -0,0 +1,160 @@
+import logging
+from cumulusci.cli.config import CliConfig
+from cumulusci.core.utils import import_class
+from cumulusci.core.exceptions import TaskNotFoundError
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.core.config import TaskConfig
+from robot.libraries.BuiltIn import BuiltIn
+from simple_salesforce import Salesforce
+
+class CumulusCI(object):
+    """ Library for accessing CumulusCI for the local git project
+
+        This library allows Robot Framework tests to access credentials to a
+        Salesforce org created by CumulusCI, including Scratch Orgs.  It also
+        exposes the core logic of CumulusCI including interactions with the 
+        Salesforce API's and project specific configuration including custom
+        and customized tasks and flows.
+
+        Initialization requires a single argument, the org name for the target
+        CumulusCI org.  If running your tests via cci's robot task (recommended),
+        you can initialize the library in your tests taking advantage of the
+        variable set by the robot task:
+        | ``*** Settings ***``
+        |
+        | Library  cumulusci.robotframework.CumulusCI  ${ORG}
+
+    """
+
+    ROBOT_LIBRARY_SCOPE = "GLOBAL"
+
+    def __init__(self, org_name=None):
+        if not org_name:
+            org_name = 'dev'
+        self.org_name = org_name
+        self.sf = self._init_api()
+        self.tooling = self._init_api('tooling/')
+        # Turn off info logging of all http requests 
+        logging.getLogger('requests.packages.urllib3.connectionpool').setLevel(logging.WARN)
```

Comment:
i don't love this global side effect in this class, and I really really don't love it happening in the initializer. can this be handled in the logging setup for cci?

Reply:
It's only needed in the context of robot.  Without this line, robot outputs every HTTPS call made which pollutes the logs.  cci never seemed to have this issue, perhaps because its logger doesn't log anything from other packages.